examples/app.py

```diff
@@ -85,7 +88,7 @@
     CELERY_EAGER_PROPAGATES_EXCEPTIONS=True,
     CELERY_RESULT_BACKEND="cache",
     DEBUG=True,
-    SEARCH_UI_SEARCH_API='invenio_search_ui.api',
+    SEARCH_UI_SEARCH_API='invenio_records_rest.recid_list',
```

Comment:
This won't work I'm quite sure. ``invenio_records_rest.recid_list`` exists in the API Flask application, hence a ``url_for`` running in the UI Flask application won't be able to generate URLs for the API application.